content/en/sheets/guide/rum_analysis.md

```diff
@@ -0,0 +1,85 @@
+
+## Creating a table in Sheets
+
+To start your analysis, create a table of RUM sessions in Datadog Sheets:
+
+1. Start from the RUM Sessions Explorer.
+1. Build a query to filter sessions with specific characteristics. For instance, to focus on user sessions that have encountered two or more errors, use the following query:
+     ```
+     @session.error.count:>=2 @session.type:user
+     ```
+1. Click on **Open in Sheets** to create a new table with your filtered RUM sessions data.
+
+## Uploading data to Reference Tables
+
+To enrich your RUM sessions data with additional metadata, such as identifying premium users or associating users with specific teams, use Reference Tables. Upload external data with the following steps:
+
+1. Prepare a CSV file containing additional metadata. For example, your file might include columns for user IDs, premium status, and team information.
+1. Navigate to [Reference Tables][1] and click **New Reference Table +**.
+1. Upload the CSV file and define the primary key for the table.
+
+For more information, see the [Reference Tables][2] documentation.
+
+## Enriching RUM data with lookups
+
+1. In your Sheets interface, click **Add Lookup** at the top of the page.
+1. Select the Reference Table you uploaded and specify the common column, such as User ID, to match records.
+1. The Lookup function adds the relevant metadata from the Reference Table to your RUM sessions table.
+
+## Using pivot tables for analysis
+
+Pivot tables help you summarize and organize large datasets to find patterns and trends.
+
+1. From an existing spreadsheet that already has a table of data, click **Add Pivot Table**.
+2. In the **Rows** section, select the dimensions you want to analyze, such as the User name.
+3. In the **Calculations** section, choose the dimensions for calculations, like sum, average, count, min, and max.
+
+## Example use case: Analyzing premium user errors
+
+You have a web application and want to understand how errors are impacting your application's premium users. Follow these steps to analyze RUM sessions, enrich your data with additional user information, and gain insights into how your premium users are impacted by errors. Use sheets to help you make data-driven decisions to improve user experience and address issues more effectively.
+
+
+### Filter RUM Sessions
+1. Start from the [RUM Sessions Explorer][3] and build a query to filter sessions with two or more errors.
+
+     ```
+     @session.error.count:>=2 @session.type:user
+     ```
+     {{< img src="/sheets/guide/rum_analysis/rum_sessions_explorer.png" alt="RUM Session Explorer showing a query filtering @session.error.count:>=2 and @session.type:user" style="width:80%;" >}}
+1. Click **Open in Sheets** to create a table with the filtered sessions.
+
+### Enrich Data
+1. Upload a CSV file to a Reference Table with additional user metadata (such as premium status or team).
```

Comment:
I think this needs more explanation than is given. What is the step by step process?

Reply:
I list out the step by step above under Uploading data to Reference Tables, but I can reiterate here as well.

Reply:
You could also link up to it if you want :)

Reply:
```suggestion
1. [Upload a CSV file to a Reference Table](#uploading-data-to-reference-tables) with additional user metadata (such as premium status or team).
```